xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/layouts/LivedataLayout.vue

```diff
@@ -34,6 +34,9 @@
 -->
 <template>
   <div class="livedata-layout">
+    <span class="description" :id="descriptionId" v-if="hasDescription">
```

Comment:
It looks odd to me to place a `span` inside a `div`, this is not something we usually do in XWiki. In particular, I think this element should be a block element so a `p` would be much more appropriate. Also, in the proposal you mentioned using a `caption` element for the table layout, why did you decide to deviate from your proposal?

Reply:
This is making the location of the description more stable.
I realized that if the location of the description was up to the layout, switching layout could mean having the description moving quite radically to a new location, which can be confusing.

Reply:
Also, this is less work for implementors of new layout, as they don't have to remember to take into account the description.